### ISSUE TITLE
Support RGBA colors for opacity

### ISSUE THREAD

**Issue by ScreamZ** [diegomura/react-pdf#681]
In order to make a TAG, I would like to use opacity on a background color, but not in the text color.

Currently I can only use opacity, which result of the full container being applied with opacity.

Could be nice to use RGBA colors 

Thanks :)

**Comment by ditiem** [comment]
I would like react-pdf to understand #rgba colors. By looking at the code it seems not so hard to "fix". 

My idea would the following (in the file Base.js):

```
drawBackgroundColor() {
    const { backgroundColor, opacity = 1 } = this.style;
    const { left, top, width, height } = this.getAbsoluteLayout();

    if (backgroundColor) {
      this.root.instance.save();

      this.clip();

      this.root.instance
        .fillOpacity(colorOpacity(backgroundColor, opacity)) // <---- HERE
        .fillColor(backgroundColor.slice(0,6)) // <--- be sure we only use the RGB part
        .rect(left, top, width, height)
        .fill()
        .restore();
    }
  }
```

colorOpacity would return the opacity of backgroundColor or opacity if backgroundColor is in #rgb format.

Any collaborator could give some guidance/tips/advice?

Thanks

**Comment by diegomura** [comment]
Added and fixed in 2.0. Check: https://github.com/diegomura/react-pdf/pull/617/commits/f616ad581cd29086cf459e673a5b1a38d25f686a
